src/components/@shared/Onboarding/Header.module.css

```diff
@@ -30,12 +31,24 @@
   color: var(--brand-white);
 }
 
-.container svg {
+.container > svg {
   display: none;
 }
 
+.hideButtonContainer {
+  position: absolute;
+  display: flex;
+  justify-content: center;
```

Comment:
comment(non-blocking): this might not be needed here, as padding + aspect-ratio should be enough to center the single item